graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/shared/DirectivesUtils.kt

```diff
@@ -46,7 +46,7 @@ fun applyDirectivesKotlin(directives: List<Directive>, config: CodeGenConfig): M
             if (argumentMap.containsKey(ParserConstants.REASON)) {
                 annotations.add(deprecatedAnnotation((argumentMap[ParserConstants.REASON] as StringValue).value))
             } else {
-                throw IllegalArgumentException("Deprecated requires an argument `${ParserConstants.REASON}`")
+                annotations.add(deprecatedAnnotation("Deprecated."))
```

Comment:
I think a String like "Deprecated in GraphQL schema" might be a bit better.

Reply:
PR updated.